bpf/process/bpf_exit.c

```diff
@@ -43,10 +43,28 @@ char _license[] __attribute__((section("license"), used)) = "Dual BSD/GPL";
  * we are the last one of the thread group.
  */
 __attribute__((section("kprobe/acct_process"), used)) int
-event_exit(struct pt_regs *ctx)
+event_exit_acct_process(struct pt_regs *ctx)
```

Comment:
While you are it Jiri would it be nice if you can rename it to tg_kp_exit_acct_process and the other one tg_kp_exit_disassociate_ctty ?  I will add later some detect kprobes load / disable policies so just in time for those ;-)  (I can do it later too...)

Reply:
hum, so we want all programs with tg_kp_ prefixes?

Reply:
I thin `tg_` should suffice. The general idea is to have a (reasonable) way to have a common prefix for tetragon objects so that we can identify them in a system (e.g., when doing `bpftool prog list`).

Reply:
So yeh tg_ we do want this, for the extra kp_ up to you no strong opinion, we can go without it.
I'm working on tracing kprobes arm disarm code so being able to identify with tg_ prefix is a quicky way to say if tetragon probes are being disarmed